colosseum/declaration.py

```diff
@@ -536,10 +598,8 @@ def __str__(self):
         non_default = []
         for name in _CSS_PROPERTIES:
             try:
-                non_default.append((
-                    name.replace('_', '-'),
-                    getattr(self, '_%s' % name)
-                ))
+                getattr(self, '_%s' % name)
+                non_default.append((name.replace('_', '-'), getattr(self, name)))
```

Comment:
I feel like you've explained this one before, but I can't remember (or find the comment) - why is the double lookup needed here? At the very least, a code comment is needed to clarify.

Reply:
No longer needed. Fixed!